packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -454,9 +454,7 @@ export abstract class Axis<
         this.updatePosition();
         this.updateSelections();
 
-        this.tickLineGroup.visible = this.tick.enabled;
```

Comment:
Is this an intentional change? If it's a bug being fixed, we should make sure to capture that in JIRA for David to review.

Do the non-cartesian axes types still behave correctly after this change?

Reply:
It was being overriden in cartesianAxis update function but I have updated polarAxis to have it so there are no regression, none of the tests flagged an issue though